agent/metadata.go

```diff
@@ -15,6 +17,11 @@ import (
 // (see https://github.com/firecracker-microvm/firecracker/blob/main/docs/mmds/mmds-user-guide.md#version-2)
 const MmdsAddress = "169.254.169.254"
 
+const nexEnvSandbox = "NEX_SANDBOX"
+const nexEnvVmID = "NEX_VMID"
+const nexEnvNodeNatsHost = "NEX_NODENATSHOST"
```

Comment:
Might want to use underscores so it's more human-readable. It isn't a big deal now but we could end up having a lot more of these env vars later. `NEX_NODE_NATS_HOST` is a lot easier on the eyes.

Reply:
good call